pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+> For bugfixes
+
+Fixes #issue
+
+Related to #another_issue/PR
+
+# Problem
+- Describe problem
+- Include media if necessary
+
+# Changes
+- Describe changes
+- Include concise code blocks if necessary
+- Include media if necessary
+
```

Comment:
For both bugs and features, please include a `# Testing` section that asks to provide evidence of testing (e.g. screenshots, screen recordings, logs).

For bugs suggest to provide recordings before and after the fix to show you were able to reproduce and then fix the issue